tests/snippets/builtin_all.py

```diff
@@ -6,5 +6,5 @@
 assert all([])
 assert not all([False, TestFailingBool()])
 
-assert_raises(RuntimeError, lambda: all(TestFailingIter()))
-assert_raises(RuntimeError, lambda: all([TestFailingBool()]))
+assert_raises(RuntimeError, all, TestFailingIter())
```

Comment:
Note that this behavior is slightly different than before. Before the TestFailingIter was create when the lambda function was invoked. Now, it is instantiated when the assert_raises function is called, so any exception in the constructor will not be catched.

Reply:
Because this code is testing `all()`, i think it is fine. But it seems some other changes may bring a problem in same way.

Let me know if you want to  adapt more conservative replacement. I used regex replacement for most of the changes, so there can be some points i missed. (though passing all the tests implies there is no exceptions tests couldn't catch)